bin/command.php

```diff
@@ -630,7 +630,9 @@ private static function parse_docblock( $docblock ) {
 					$key = key( $ret['parameters'][ $param_name ] );
 					reset( $ret['parameters'][ $param_name ] );
 					if ( ! empty( $ret['parameters'][ $param_name ][ $key ][2] )
-						&& '{' === substr( $ret['parameters'][ $param_name ][ $key ][2], -1 ) ) {
+						&&
+						in_array( substr( $ret['parameters'][ $param_name ][ $key ][2], -1 ), [ '{', ',' ], true )
```

Comment:
What will happen if a multi-line comment doesn't end with a comma?

Reply:
Hi @wojtekn 

Thank you for your feedback and suggestion, I’ve updated the logic and refined the code based on it. I've also created a brief video to explain the updates, the assumptions made, and the approach taken. 

Here’s the link to the video: [Link](https://drive.google.com/file/d/1AFqszLTjLSVwp0iyU_knKTYsCAy0nQQ2/view?usp=drive_link)